Libraries/LibWeb/CSS/Length.cpp

```diff
@@ -39,6 +40,28 @@ float Length::relative_length_to_px(const LayoutNode& layout_node) const
         return m_value * layout_node.font_size();
     case Type::Rem:
         return m_value * layout_node.document().document_element()->layout_node()->font_size();
+    case Type::Vw:
+        return layout_node.document().frame()->viewport_rect().width() * (m_value / 100);
+    case Type::Vh:
+        return layout_node.document().frame()->viewport_rect().height() * (m_value / 100);
+    case Type::Vmin: {
+        auto viewport = layout_node.document().frame()->viewport_rect();
```

Comment:
```suggestion
        auto& viewport = layout_node.document().frame()->viewport_rect();
```

Reply:
This change would result in an error and I sadly don't have the expertise to dig further into the code

Reply:
Ah, right - try this then:

```suggestion
        const auto& viewport = layout_node.document().frame()->viewport_rect();
```

Reply:
That compiles, however if it's not really necessary I would spare myself having to deal with squashing (I can also just make a new commit however that's probably a little too atomic)

Reply:
It's not the end of the world to copy this rect. If it turns out to be an issue someday in the future, we can easily tweak it :)

Reply:
Alright :)